src/styles/blocks/header.scss

```
@@ -0,0 +1,66 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  font-weight: 500;
+}
+
+.nav {
+  &__item {
+    text-decoration: none;
+
+    margin-left: 20px;
+    color: $color-nav-link;
+    font-size: 12px;
+    list-style: none;
+  }
+  &__item:first-child {
+    margin-left: 0;
+  }
+
+  &__list {
+    display: flex;
+    list-style: none;
+    padding: 0;
+    margin-bottom: 0;
+    margin-top: 0;
+  }
+
+  &__link {
+    display: flex;
+    align-items: center;
+    text-decoration: none;
+    text-align: center;
+    color: $color-nav-link;
+    text-transform: uppercase;
+    height: 60px;
+  }
+
+  &__link:hover {
+    color: $color-blue-accent;
+  }
```

use scss nesting. Fix it everywhere
```suggestion
  &__link {
    display: flex;
    align-items: center;
    text-decoration: none;
    text-align: center;
    color: $color-nav-link;
    text-transform: uppercase;
    height: 60px;
    
    &:hover {
      // write styles here
    }
  }
```